### PR TITLE
Add r2pm man page

### PR DESCRIPTION
**Pull request by isra17**
Hey,
I'm looking for some way to start contributing to radare2, hopefully this can be helpful.

Closes radare/radare2#3398

### CODE REVIEW

**Comment by crowell** [comment]
probably should have an example `r2pm -s` with no search term to show how to list all available plugins.

File: man/r2pm.1 [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.Dd Oct 1, 2015
+.Dt R2PM 1
+.Sh NAME
+.Nm R2PM
+.Nd The radare2 plugins manager
```
Comment (crowell) [inline man/r2pm.1]: radare2 package manager

**Comment by isra17** [comment]
I did address your comments. Let me know if it's all good.

**Comment by crowell** [comment]
merged

**Pull request closed without merging**